app/views/prompts/_prompt_blurb.html.erb

```diff
@@ -1,5 +1,5 @@
 <% # expects "prompt", if index is passed will use that in title if none exists, if suppress_claims is true will ignore %>
-<li class="<% if is_author_of?(prompt) %>own <% end %>prompt blurb group" role="article">
+<li class="<% if is_author_of?(prompt) %>own <% end %> blurb group" role="article">
```

Comment:
This could probably be changed to have less erb verbiage, e.g.:
```
<% is_author_of?(prompt) ? "own" : "" %>
```
(or the same thing with an `if` statement